### ISSUE TITLE
Multiple apps single project :- i.e Customer, 3rd Party and Admin 

### ISSUE THREAD

**Issue by ben700** [firebase/firebase-tools#6003]
Think of Uber Eats, you have Rider App, Driver App and 3rd Party (takeaway) App

So I have three apps but all sit within one firebase project, one set of database rules, and one set of functions 

Maybe I'm doing wrong but I want each target to also have it's own app and therefore it's own build

**Comment by ben700** [comment]
No, think of Uber. You have the Uber Rider App (Android, ios and web) you
also have the driver app (Android, ios and web) and for Uber eats you also
have the restaurant app (Android, ios and web). They have some common
functionality like object model but they are three separate apps each with
own app in play or iStore and hosted on their own URL

I'd also have separate environments or flavours i.e.  dev, test and
production

Ben

**Comment by aalej** [comment]
Apologies, but I'm still not getting a clear picture of your desired feature. With this, could you explain in detail the exact feature related to firebase-tools that you want to be implemented? 

Just to share, if you are connecting multiple build variants that could share the same Firebase resources, you can register the variants with the same Firebase project. However, if you have multiple build variants that are based on release status, it is recommended that you register each variant with a separate Firebase project as mentioned [here](https://firebase.google.com/docs/projects/dev-workflows/general-best-practices#registering-app-variants).
